tox.ini

```diff
@@ -32,11 +32,12 @@ commands = pytest -v check_sphinx.py {posargs}
 description = invoke sphinx-build and try to build the HTML page (also check link validity)
 
 [testenv:flakes]
-deps = pytest-flakes >= 0.2
-       pytest-pep8
-description = run static analysis and style check using flakes and pep-8
-commands = pytest --flakes -m flakes tox tests
-           pytest --pep8 -m pep8 tox tests
+basepython=python3.6
+deps = flake8 == 3.4.1
+       flake8-bugbear == 17.4.0
+description = run static analysis and style check using flakes
+commands = python -m flake8 --show-source tox setup.py {posargs}
```

Comment:
2 calls are no longer needed due to tool convergence
i believe you can also directly invoke flake8 without pytohn -m